pkgs/by-name/xr/xrescat/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  gnumake,
+  gcc,
+  gdb,
+  xorg,
+  SDL2,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xrescat";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "regolith-linux";
+    repo = "xrescat";
+    rev = "e8e261441682244112b2020e2ad102768e6ba3fd";
```

Comment:
There is a release from february (v1.2.1-3), why not use it?
If this revision contains something that made you opt for it, please add a comment to denote it